liquibase/changeLogs/capture/tables/datumConvertedValues/datumConvertedValues.sql

```diff
@@ -0,0 +1,12 @@
+create table ${AQTS_SCHEMA_NAME}.datum_converted_values
+(datum_converted_values_id                int8 NOT NULL GENERATED BY DEFAULT AS IDENTITY,
+json_data_id                              int8 NOT NULL,
```

Comment:
Rather than `int8` on lines 2 and 3, I think the other tables use `bigint` instead.  It's unclear to me if `int8` is the same thing but the documentation seems to imply it?

https://www.postgresql.org/docs/11/datatype-numeric.html

Also, let's add a `(start with 1)` to line 2 after `IDENTITY` - see this other table for an example: https://github.com/usgs/aqts-capture-db/blob/master/liquibase/changeLogs/capture/tables/fieldVisitReadings/fieldVisitReadings.sql#L3

Also I think we can kill all the `NOT NULL` declarations in this file.